src/frontend/screens/Library/index.tsx

```diff
@@ -189,6 +190,18 @@ export default React.memo(function Library(): JSX.Element {
 
   const [showCategories, setShowCategories] = useState(false)
 
+  const [isAlphabetFilterVisible, setIsAlphabetFilterVisible] = useState(
```

Comment:
```suggestion
  const [showAlphabetFilter, setShowAlphabetFilter] = useState(
```

so it's consistent with the other useState calls that are named `show...` and `setShow...` above this one